fe/fe-core/src/main/java/com/starrocks/sql/analyzer/QueryAnalyzerV2.java

```diff
@@ -0,0 +1,603 @@
+                                cteRelation.getColumnOutputNames(),
+                                cteRelation.getCteQuery());
+                        newCteRelation.setAlias(tableRelation.getAlias());
+                        newCteRelation.setScope(
+                                new Scope(RelationId.of(newCteRelation), new RelationFields(outputFields.build())));
+                        return newCteRelation;
+                    }
+                }
+
+                Table table = resolveTable(tableRelation.getName());
+                if (table instanceof View) {
+                    View view = (View) table;
+                    String inlineViewDef = view.getInlineViewDef();
+
+                    StarRocksLexer lexer =
+                            new StarRocksLexer(new CaseInsensitiveStream(CharStreams.fromString(inlineViewDef)));
+                    CommonTokenStream tokenStream = new CommonTokenStream(lexer);
+                    StarRocksParser parser = new StarRocksParser(tokenStream);
+
+                    StarRocksParser.SingleStatementContext singleStatementContext = parser.singleStatement();
+                    QueryStatement stmt =
+                            (QueryStatement) new AstBuilder().visitSingleStatement(singleStatementContext);
+
+                    SubqueryRelation viewRelation = new SubqueryRelation(tableName.getTbl(), stmt.getQueryRelation());
+                    viewRelation.setAlias(tableName);
+                    return viewRelation;
+                } else {
+                    if (AnalyzerUtils.isSupportedTable(table)) {
+                        tableRelation.setTable(table);
+                        return tableRelation;
+                    } else {
+                        throw unsupportedException("unsupported scan table type: " + table.getType());
+                    }
+                }
+            } else {
+                return relation;
+            }
+        }
+
+        @Override
+        public Scope visitTable(TableRelation node, Scope outerScope) {
+            TableName tableName = node.getAlias();
+            Table table = node.getTable();
+
+            ImmutableList.Builder<Field> fields = ImmutableList.builder();
+            ImmutableMap.Builder<Field, Column> columns = ImmutableMap.builder();
+
+            for (Column column : table.getFullSchema()) {
+                Field field;
+                if (table.getBaseSchema().contains(column)) {
+                    field = new Field(column.getName(), column.getType(), tableName, null, true);
+                } else {
+                    field = new Field(column.getName(), column.getType(), tableName, null, false);
+                }
+                columns.put(field, column);
+                fields.add(field);
+            }
+
+            node.setColumns(columns.build());
+            session.getDumpInfo().addTable(node.getName().getDb().split(":")[1], table);
```

Comment:
Is `split(":")[1]` safe?

Reply:
This is not a newly written code, but a copy of the old code. This style is really not very good, but it is safe. Because after being analyzed, it must be in the format of xx:xx